### PR TITLE
Add the MaxConditionCost option

### PR DESCRIPTION
**Pull request by bjchambers**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace "<Jira issue #>" in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by bjchambers** [comment]
R: @davorbonaci

File: sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/CloudDebuggerOptions.java [hunk 1]
```diff
@@ -30,15 +30,22 @@
...
   void setEnableCloudDebugger(boolean enabled);
 
-  @Description("The Cloud Debugger debugee to associate with. This should not be set directly.")
+  /** The Cloud Debugger debuggee to associate with. This should not be set directly. */
+  @Description("The Cloud Debugger debuggee to associate with. This should not be set directly.")
   @Hidden
   @Nullable Debuggee getDebuggee();
   void setDebuggee(Debuggee debuggee);
+
+  /** The maximum cost (as a ratio of CPU time) allowed for evaluating conditional breakpoints. */
```
Comment (davorbonaci) [inline sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/CloudDebuggerOptions.java]: Specify minimum and maximum and what they mean.

**Comment by davorbonaci** [comment]
LGTM

File: sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/CloudDebuggerOptions.java (hunk 1 above)
Comment (bjchambers) [inline sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/CloudDebuggerOptions.java]: Done, I think. PTAL?

**Pull request closed without merging**